examples/UbidotsGetValue/UbidotsGetValue.ino

```diff
@@ -1,38 +1,47 @@
-#include <UbidotsArduinoGPRS.h>
-#include <SoftwareSerial.h> 
-
-//Serial Relay - Arduino will patch a 
-//serial link between the computer and the GPRS Shield
-//at 19200 bps 8-N-1
-//Computer is connected to Hardware UART
-//GPRS Shield is connected to the Software UART 
-#define APN "Put_the_APN_here" // Assign the APN 
-#define USER "Put_the_APN_user_herer"  // If your apn doesnt have username just put ""
-#define PASS "Put_the_APN_pwd_here"  // If your apn doesnt have password just put ""
-#define TOKEN "Put_your_Ubidots_token_here"  // Replace it with your Ubidots token
-#define DEVICE_LABEL "Put_your_device_API_label" // Assign your device api label
-#define VARIABLE_LABEL "Put_your_variable_API_label" // Assign your device api label
-
-Ubidots client(TOKEN);
-SoftwareSerial gprs = SoftwareSerial(7, 8);
-SoftwareSerial *GPRSSerial = &gprs;
+// This example retrieves last value of a variable from the Ubidots API
+// using TCP protocol.
+
+/****************************************
+ * Include Libraries
+ ****************************************/
+#include <Ubidots.h>
+
+/****************************************
+ * Define Constants
+ ****************************************/
+// Your GPRS credentials, if any
+const char *APN  =           "Put_the_APN_here";
+const char *USER =           "Put_the_APN_user_here";
+const char *PASS =           "Put_the_APN_pwd_here";
+const char *TOKEN=           "Put_your_Ubidots_token_here";
```

Comment:
there must be a blank space after a variable, please refer to Google C++ style guide https://google.github.io/styleguide/cppguide.html#Variable_and_Array_Initialization

This observations applies to ALL the examples